src/shell.js

```diff
@@ -34,7 +34,8 @@ var Module = typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {
 #endif // USE_CLOSURE_COMPILER
 
 #if POLYFILL
-#if WASM_BIGINT && MIN_SAFARI_VERSION < 150000
+#if WASM_BIGINT && MIN_SAFARI_VERSION < 140100
+// TODO(features): Fix this back to 150000
```

Comment:
Can you explain this a little?

Reply:
Like in a comment you mean?
The change here is just to make sure we don't emit the polyfill during the time when we are pretending to be targeting 14.1 by default but are actually emitting bigint. Once everything land this will go back to how it was.

Reply:
Yes I think I get it now.. but the comment could perhaps use more context.  Perhaps link to this bug?  Or mention why this is artificially lowered

Reply:
I filed https://github.com/emscripten-core/emscripten/issues/23184 and linked to it.